processor/probabilisticsamplerprocessor/metadata.yaml

```
@@ -28,3 +28,10 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    processor_probabilistic_sampler_count_spans_sampled:
+      enabled: false
+      description: Count of spans that were sampled or not
+      unit: "1"
```

```suggestion
      unit: "{spans}"
```

I know you used the other ones as the base, but they are wrong (probably my fault 🙈 )